build.assets/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN (mkdir -p /opt/protoc && \
      unzip -d /opt/protoc /tmp/${TARBALL} && \
      go get -u github.com/gogo/protobuf/proto github.com/gogo/protobuf/protoc-gen-gogo github.com/gogo/protobuf/gogoproto \
      github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
-     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger)
+     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-openapiv2)
```

Comment:
Hmm, looks like this is still susceptible to similar breakages in the future since it is not using pinned versions/branches. I think a better fix here it to backport changes from [here](https://github.com/gravitational/gravity/pull/763/files#diff-1e5924d92807f9242cbbcdd715eb80a2237f28d8f4642a1d987df5b9c04a0811)